src/utils/address_translation.ml

```diff
@@ -11,5 +11,5 @@ let generate_tid_map prog =
     inherit [addr Tid.Map.t] Term.visitor
     method enter_term _ t addrs = match Term.get_attr t address with
       | None -> addrs
-      | Some addr -> Map.add addrs ~key:(Term.tid t) ~data:addr
+      | Some addr -> Map.add_exn addrs ~key:(Term.tid t) ~data:addr
```

Comment:
I believe this should never blow up, otherwise the project is very broken anyways.